File: Programming_Basics_with_Python/conditional_statements_exercise/02._bonus_score.py
score = int(input())
extra_bonus_score = 0
if score <= 100:
    bonus_score = 5
elif 1000 > score > 100:
    bonus_score = score * 0.2
else:
    bonus_score = score * 0.1
if score % 2 == 0:
    extra_bonus_score = 1
elif score % 10 == 5:
    extra_bonus_score = 2
total_bonus_score = bonus_score + extra_bonus_score
total_score = score + bonus_score + extra_bonus_score
print(total_bonus_score)
print(total_score)

File: Programing_Basics_with_JS/conditionalStatementsExercise/02.BonusScore.js
function bonusScore(input) {
    let number = Number(input[0]);

    let bonusPoint = 0;

    if (number <= 100) {
        bonusPoint += 5;
    } else if (number > 100 && number < 1000) {
        bonusPoint += number * 0.2;
    } else {
        bonusPoint += number * 0.1;
    }

    if (number % 2 === 0) {
        bonusPoint += 1;
    } else if (number % 10 === 5) {
        bonusPoint += 2;
    }

    console.log(bonusPoint);
    console.log(bonusPoint + number);
}

bonusScore(["20"])
bonusScore(["175"])
bonusScore(["2703"])
bonusScore(["15875"])